supporting_content/metadata/metadata.md

```diff
@@ -0,0 +1,131 @@
+
+    Method: GET 
+    Endpoint : <api_root>/FOCUS/metadata/data_generator
+####
+
+#### Response
+```
+{
+	"DataGenerator": "awesome_corp",
+	"ProviderColumnPrefix": "awecorp
+	"ProviderTagPrefixes": ["awecorp", "ac"]
+}`
+```
+
+## Example Schema Metadata
+
+In this example, the billing data includes two different structures of data. An older schema for data previously provided, and a newer schema for more recent data. Both schemas metata is provided. 
+
+### API
+
+#### Endpoint: <api_root>/FOCUS/metadata/schemas 
+#### Example Request:
+    endpoint: <api_root>/FOCUS/metadata/schemas 
+    method: GET 
+####
+
+#### Response 
+```
+{
+	[
+		{
+			"SchemaID": "1234",
+			"FocusVersion": "1.0",
+			"name": "my original schema",
+			"CreationDate": "2024-01-01T12:01:03.083z"
+			"schema_column_endpoint": <api_root>/FOCUS/metadata/schemas/1234/columns
+		},
+		{
+			"SchemaID": "2345",
+			"FocusVersion": "1.1",
+			"name": "my new schema",
+			"CreationDate": "2024-07-01T12:00:04.001z"
+			"schema_column_endpoint": <api_root>/FOCUS/metadata/schemas/2345/columns
+		}
+	]
+}
+```
+
+## Example Schema Column Definition Metadata
+
+In this example the billing data generator's FOCUS metadata API is queried for the schema_id = 1234. 
+
+### API Schema 
+
+#### Endpoint: <api_root>/FOCUS/metadata/schema/{id}/columns
+#### Example Request:
+
+    Method: GET 
+    Endpoint : <api_root>/FOCUS/metadata/schema/1234/columns
+####
+
+#### Response 
+```
+{
+	"SchemaID": "1234",
+	"FOCUS_version": "1.0",
+    "name": "my original schema",
+    "CreationDate": "2024-01-01T12:01:03.083z"
+	"ColumnDefinition": [
+                 {
+                     ColumnName: "BillingAccountId",
+                     DataType: "STRING"
+                     StringMaxLength: 64,
+                     StringEncoding: "UTF-8"
+                 },
+                 {
+                     ColumnName: "BillingAccountName",
+                     DataType: "STRING"
+                     StringMaxLength: 64,
+                     StringEncoding: "UTF-8"
+                 },
+                 {
+                     ColumnName: "ChargePeriodStart",
+                     DataType: "DATETIME"
+                 },
+                 {
+                     ColumnName: "ChargePeriodEnd",
+                     DataType: "DATETIME"
+                 },
+                 {
+                     ColumnName: "BilledCost",
+                     DataType: "DECIMAL",
+                     NumericPrecision: 20,
+                     NumberScale: 10
+                 },
+                 {
+                     ColumnName: "EffecitiveCost",
+                     DataType: "DECIMAL",
+                     NumericPrecision: 20,
+                     NumberScale: 10
+                 },
+         ]
+}`
+```
+
+## Example Schema Reference Metadata
+
+In this example, when the provider returns the FOCUS data they include in the response a reference to the schema utilized in the FOCUS dataset. 
```

Comment:
Could be also useful to include the DataGeneratorVersion used to generate this FOCUS dataset?

Reply:
Agreed to merge this PR and open a related issue for the version vs schemaId discussion